monitoring/metrics.rb

```diff
@@ -21,10 +21,12 @@ def create_metric_descriptor project_id:
   # [END monitoring_create_metric]
 end
 
-def delete_metric_descriptor descriptor_name:
+def delete_metric_descriptor project_id:, descriptor_name:
   # [START monitoring_delete_metric]
   client = Google::Cloud::Monitoring::Metric.new
```

Comment:
can you include comments describing project_id and descriptor_name?

Reply:
Hi,

I'd be glad to add some more documentation. All the other methods have no comments describing their parameters. Do you have any recommendation regarding formatting of this documentation?

As for the comments themselfves, all I could come up with is:
```ruby
# project_id: the text identifer of you Google Cloud project
# descriptor_name: the text name of the descriptor (eg: "run.googleapis.com/request_count")
def delete_metric_descriptor descriptor_name:
```

Should I add the same kind of comment to the other functions?

Reply:
That looks great. Just make sure they're inside the method and region tags

Reply:
✅ Done : https://github.com/GoogleCloudPlatform/ruby-docs-samples/pull/492/commits/0ebdcff60585116c27a658601059df6ade8ecfb7